agent/src/main/java/com/teamscale/jacoco/agent/commit_resolution/git_properties/GitMultiProjectPropertiesLocator.java

```diff
@@ -22,7 +22,7 @@
  */
 public class GitMultiProjectPropertiesLocator implements IGitPropertiesLocator {
 
-	private final Logger logger = LoggingUtils.getLogger(GitSingleProjectPropertiesLocator.class);
+	private final Logger logger = LoggingUtils.getLogger(GitMultiProjectPropertiesLocator.class);
```

Comment:
To avoid similar issues the future, we should (as we already in most cases) use `LoggingUtils.getLogger(this)` except for the cases where we create the logger from a static context. Can you fix the other "mis-use" in `GitSingleProjectPropertiesLocator` as well?